app/views/users/sessions.py

```diff
@@ -141,6 +146,77 @@ def add_speaker_view(event_id, session_id):
         return redirect(url_for('.index_view', event_id=event_id))
 
 
+@event_sessions.route('/download_speakers_as_csv/')
+@belongs_to_event
+@can_access
+def download_speakers_as_csv(event_id):
+    sessions = DataGetter.get_sessions_by_event_id(event_id)
+    event = DataGetter.get_event(event_id)
+    main = []
+    data = ["Speaker Name", "Speaker Session(s)", "Speaker Email", "Speaker Status", \
+            "Speaker Mobile", "Speaker Organisation", "Speaker Position"]
+    main.append(data)
+    for session in sessions:
+        for speaker in session.speakers:
+            data = []
+            data.append(speaker.name) if speaker.name else data.append('')
```

Comment:
@LuD1161 and this would also solve the codacy issue.  You can click on the codacy issue to read more about why the issue appears and check google.